libs/books/feature/src/lib/reading-list/reading-list.component.scss

```diff
@@ -40,6 +40,11 @@
   white-space: nowrap;
 }
 
+.reading-list-actions {
+  display: flex !important;
+  flex-direction: row !important;
```

Comment:
Why the use of `!important` here? Could better specificity help? Or more proper handling of the rules?